app/main.py

```diff
@@ -1 +1,10 @@
-#  write your code here
+from typing import List, Union
+
+from app.animals.animal import Animal
+from app.animals.cat import Cat
+from app.animals.dog import Dog
+
+
+def feed_animals(all_animals: List[Union["Animal", "Cat", "Dog"]]) -> int:
+    all_food = sum(animal.feed() for animal in all_animals)
+    return all_food
```

Comment:
You can put it directly in return to avoid creating a new variable that's used only once